apps/mobile/src/components/TabNavivation/TabBar.tsx

```diff
@@ -1,14 +1,37 @@
 import type { FC } from 'react';
-import type { ViewStyle } from 'react-native';
+import type { ImageSourcePropType, ViewStyle } from 'react-native';
 import { StyleSheet, View } from 'react-native';
 import type { BottomTabBarProps } from '@react-navigation/bottom-tabs';
+import { useDrawerStatus } from '@react-navigation/drawer';
+import type { DrawerNavigationHelpers } from '@react-navigation/drawer/lib/typescript/src/types';
 import type { Route } from '@react-navigation/native';
+import { appActions, appState } from 'state/app';
+import { nativeAsset } from 'utils/config';
```

Comment:
don't use nativeAsset directly like this, import `assets` from `@walless/ioc` instead.